pages/how-to/how-to-create-tradingview-charts.mdx

```diff
@@ -0,0 +1,116 @@
+import { Tab, Tabs } from "nextra-theme-docs";
```

Comment:
moved from documentation to the how-to section.

Reply:
I think we should update the "benchmarks" section of the docs to talk about how the API supports charting via tradingview, then link users here for how to do it.